client/components/header/styles.js

```diff
@@ -82,7 +82,7 @@ export const MobileMenuPageOverlay = styled.div`
   left: 0;
   height: 100%;
   width: 100%;
-  z-index: 500;
+  z-index: 700;
   background: ${props => props.theme.colors.dark};
```

Comment:
According to the design, the overlay no longer has any background color.
background-color and opacity should probably be removed :)

Reply:
That makes sense! That’s why I put the overlay in front... because the menu looked weird when it wasnt the same shade as the top bar. 

I’ll make the changes. Thanks!